### PR TITLE
dependencies up-to-date + no global execution context 

### PR DESCRIPTION
**Pull request by MasseGuillaume**
- source map in resources

### CODE REVIEW

**Comment by MasseGuillaume** [comment]
@lihaoyi ping ?

File: build.sbt [hunk 1]
```diff
@@ -26,17 +26,20 @@ val example = crossProject.settings(
...
 val exampleJVM = example.jvm.settings(
-  (resources in Compile) += {
-    (fastOptJS in (exampleJS, Compile)).value
-    (artifactPath in (exampleJS, Compile, fastOptJS)).value
+  (resources in Compile) ++= {
+    def andSourceMap(aFile: java.io.File) = Seq(
+      aFile,
+      file(aFile.getAbsolutePath + ".map")
+    )
+    andSourceMap((fastOptJS in (exampleJS, Compile)).value.data)
```
Comment (stewSquared) [inline build.sbt:43]: @MasseGuillaume Could you explain this to me?

**Comment by stewSquared** [comment]
This PR has grown a bit stale. We'll be upgrading to akka-http.

**Comment by stewSquared** [comment]
Upgrade to akka-http done in #45
@MasseGuillaume Please let me know if there was anything else from this PR you'd like to update.

**Pull request closed without merging**